docs/docs/outputs/logging.md

```diff
@@ -1,57 +1,100 @@
-# Tracee Logging Examples
+# Defining Tracee Logs
```

Comment:
similar to the other suggestion, the word "defining" is confusing, can remove it